flumeDevice

```diff
@@ -248,25 +269,36 @@ def getNotifCount()
 def fetchUsageAlerts()
 {
     def tempCount = getAlertCount() ?: 0
-    
+
     def queryString = "?limit=100&offset=${tempCount}&sort_direction=ASC&flume_leak=true"
     def resp = httpExec("GET", genParamsMain("users/${getUserId()}/usage-alerts/${queryString}"), true)?.data
     logDebug("fetchUsageAlerts resp = ${resp}")
-    
+
     if(resp.data)
     {
         for(alert in resp.data)
         {
             tempCount++
-                
-            // check for Flume Smart Leak alerts            
+
+            // check for Flume Smart Leak alerts
             if(true == alert.flume_leak)
             {
                 sendEvent(name: "water", value: "wet")
             }
-        }        
+        }
         setAlertCount(tempCount)
-    }    
+    }
+}
+
+def setLastPollTS()
+{
+    state.lastPollTS = new Date().getTime()
+}
+
+def getLastPollTS()
+{
+    lastPoll = state.lastPollTS == null ? 0 : state.lastPollTS.toInteger()
+    return lastPoll
```

Comment:
somehow, not sure, this was either `null` or a `string`, this flippy flop should resolve the error I was getting
<img width="957" alt="Screen Shot 2022-01-26 at 11 58 54 AM" src="https://user-images.githubusercontent.com/513738/151226420-378e0435-027e-4bb5-86c6-30ebf0bda2ab.png">